src/__tests__/msal-cache.test.ts

```
@@ -41,16 +74,15 @@ describe("MSAL Cache Plugin", () => {
+      const cachePlugin = await createCachePlugin();
       const deserializeMock = vi.fn();
       const cacheContext = {
         tokenCache: {
           deserialize: deserializeMock,
         },
       } as unknown as TokenCacheContext;
 
-      const consoleErrorSpy = vi.spyOn(console, "error").mockImplementation(() => {
-        // Intentionally empty to suppress console output during tests
-      });
+      const consoleErrorSpy = vi.spyOn(console, "error").mockImplementation(() => {});
```

Biome (2.4.4)

[error] Unexpected empty block.

(lint/suspicious/noEmptyBlockStatements)